src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -86,6 +87,21 @@ private void checkAuthAttributesNoDuplicates(final Map<String, List<Authorizatio
         }
     }
 
+    private void checkAuthorisationForResourceAreValid(final Resource resource)
+            throws UnableProcessException, ServiceTemporarilyUnavailableException, ForbiddenOperationException {
+
+        checkAuthAttributesAreValid(resource.getAuthorization());
```

Comment:
what's the difference between `checkAuthAttributesAreValid` and `areAllAuthorizationsForResourceValid`? I thought that the plan was to replace the former by the later, right? Since `areAllAuthorizationsForResourceValid` can perform all of the work that `checkAuthAttributesAreValid` performs. And in fact `checkAuthAttributesAreValid` only makes a number of calls to `isAuthorizationAttributeValid`. I think it's may be the source of potential security breaches if we maintain both methods and a user ends up thinking that it's safe to only `isAuthorizationAttributeValid` when in fact, `areAllAuthorizationsForResourceValid` should be also called.

Please, correct me if I missed something.